### DISCUSSION TITLE
Is there any (big) disadvantage to not download/update uniprot everytime?

### DISCUSSION THREAD

**Discussion by Unaimend** [jotech/gapseq#206] [Q&A]
Due to the fact the the update procedure takes a long time, I'd like to know if the internal versions of the DB in gapseq are updated frequently thus making the update not super necessary.

Thanks for your time

**Comment by jotech** [comment]
yes we aim to update the resources frequently. details can be found here https://zenodo.org/records/10047603
if newer sequences are available, then the download is done automatically by gapseq
in gapseq output files you can also find a hash of the sequences database
hope this helps!

**Reply by Unaimend** [reply]
Thanks for the information. Specifically, I'd like to know if it is a bad idea to use `--force offline` in gapseq find because the Uniprot download takes a long time for me and due to the fact how my internal scripts work I download gapseq for every project again

**Comment by jotech** [comment]
you mean `gapseq find -O` for offline mode?

Could work, but if the sequences are downloaded correctly once, then no download should be needed anyway.
